include/databaseFunctions.php

```diff
@@ -103,21 +78,62 @@ function DisplayPic($pic){
 			</head>
 			<body>
 				<h1>".$pic['title']."</h1>
-				<h2>by ".$pic['photographer']."</h2>
+				<h2>by ".$pic['author']."</h2>
 				<h3>date: ".$pic['date']."</h3>
 				<div>
 					<img src='".$pic['link']."'alt='".$pic['flavor']."'>
-				</div>
-				<a href='index.php'>back to home</a>
-			</body>
+				</div>";
+				if($pic['body']!=null){
+					echo"<p>".$pic['body']."</p><br>";
+				}
+		echo"</body>
 		</html>
 		";
 	}
 
 	function getNumberPics(){
 		$result = dbQuery("
-				SELECT COUNT(picID)
-				FROM pics
+				SELECT COUNT(postID)
+				FROM posts
+				WHERE isPic=1
 			")->fetch();
+		return $result['COUNT(postID)'];
+	}
+
+	//BOTH
+
+	function GetPost($postID){
+		$result = dbQuery("
+			SELECT *
+			FROM posts
+			WHERE postID = $postID
+		")->fetch();
 		return $result;
 	}
+
+	function DeletePost($postID){
+		$result = dbQuery("
+			DELETE FROM posts
+			WHERE postID = $postID
+		")->fetch();
+	}
+
+	function isPicture($post){
```

Comment:
Since all you need here is the `postId` I'd take that as a parameter instead of the whole post. That way, if someone has the post id but not the whole post, they don't have to fetch the post just to call this function.

Also, if you have the whole post, can't you just check `isPic` from that and avoid this query entirely?